Functions.Templates/Templates/BlobTrigger-Python-v2-Append/blob_trigger_template.md

```diff
@@ -0,0 +1,36 @@
+# Azure Functions: Blob Trigger in Python
+
+## Blob Trigger
+
+The Blob storage trigger starts a function when a new or updated blob is detected. The blob contents are provided as input to the function. The Azure Blob storage trigger requires a general-purpose storage account. Storage V2 accounts with hierarchical namespaces are also supported.
+
+## Using the Template
+
+Following is an example code snippet for Blob Trigger using the [Python programming model V2](https://aka.ms/pythonprogrammingmodel) (currently in Preview).
+
+```python
+import logging
+import azure.functions as func
+app = func.FunctionApp()
+@app.function_name(name="BlobTrigger1")
+@app.blob_trigger(arg_name="myblob", path="samples-workitems/{name}",
+                  connection="AzureWebJobsStorage")
+def test_function(myblob: func.InputStream):
+   logging.info(f"Python blob trigger function processed blob \n"
+                f"Name: {myblob.name}\n"
+                f"Blob Size: {myblob.length} bytes")
+```
+
+To run the code snippet generated through the command palette, note the following:
+
+- The function application is defined and named `app`.
+- Confirm that the parameters within the trigger reflect values that correspond with your storage account.
+- The name of the file must be `function_app.py`.
+  
+Note that Blob input and output bindings are also supported in Azure Functions. To learn more, see [Azure Blob storage bindings overview](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-blob?tabs=in-process%2Cextensionv5%2Cextensionv3&pivots=programming-language-python)
```

Comment:
Can you please use existing aka.ms links for this? This helps in tracking their success.